modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -81,5 +81,27 @@ Anypoint Service Mesh is installed and the status of all pods is displayed:
 +
 image::verified-service-mesh-installer-output.png[]
 
+=== Configure Headers for Client Authorization-based Policies
 
+Anypoint Service Mesh is configured with minimized whitelisted headers for client-based policies:
 
+- jwt
+- authorization
+- client_id
+- client-id
+- client_secret
+- client-secret
+
+Before use, you must whitelist a custom header. Because header whitelisting is performed at the cluster level, you do not need to whitelist headers for every managed API.
```

Comment:
```suggestion
Before use a custom header, you must whitelist it. Because header whitelisting is performed at the cluster level, you do not need to whitelist headers for every managed API.
```

Reply:
Done! Thanks!